scripts/docker-compose-mysql/jes-cromwell/jes-config/application.conf

```diff
@@ -0,0 +1,68 @@
+system.new-workflow-poll-rate=1
+
+google {
+
+  application-name = "cromwell"
+
+  auths = [
+    {
+      name = "application-default"
+      scheme = "application_default"
+    }
+  ]
+}
+
+engine {
+  filesystems {
+     gcs {
+       auth = "application-default"
+     }
+  }
+}
+
+call-caching {
+  enabled = false
+}
+
+backend {
+  default = "Jes"
+  providers {
+    Jes {
+          actor-factory = "cromwell.backend.impl.jes.JesBackendLifecycleActorFactory"
+          config {
+            // Google project
+            project = "broad-dsde-cromwell-dev"
+
+            // Base bucket for workflow executions
+            root = "gs://tjeandet-cromwell-execs"
+
+            // Polling for completion backs-off gradually for slower-running jobs.
+            // This is the maximum polling interval (in seconds):
+            maximum-polling-interval = 600
+
+            genomics {
+              // A reference to an auth defined in the `google` stanza at the top.  This auth is used to create
+              // Pipelines and manipulate auth JSONs.
+              auth = "application-default"
+              // Endpoint for APIs, no reason to change this unless directed by Google.
+              endpoint-url = "https://genomics.googleapis.com/"
+            }
+
+            filesystems {
+              gcs {
+                // A reference to a potentially different auth for manipulating files via engine functions.
+                auth = "application-default"
+              }
+            }
+          }
+    }
+  }
+}
+
+database {
+  db.url = "jdbc:mysql://mysql-db/cromwell_db?useSSL=false"
```

Comment:
Can you add `?rewriteBatchedStatements=true` to this? It won't do anything at the moment but future us will thank us for having it in there now